src/main/java/bookstore/dto/category/CreateCategoryRequestDto.java

```diff
@@ -0,0 +1,11 @@
+package bookstore.dto.category;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record CreateCategoryRequestDto(
+        @NotBlank
+        String name,
```

Comment:
Should we set size for these fields?

Reply:
OK, then I have to set @Size for CreateBookRequestDto too